src/scenes/Moves/Ppm/PaymentReview/index.jsx

```diff
@@ -22,19 +35,42 @@ class PaymentReview extends Component {
             </ProgressTimeline>
           }
         />
-        <div className="usa-grid ">
-          <ul>
-            <li>
-              <Link to={weightTicketsPage} data-cy="weight-ticket-link">
-                Weight Ticket
-              </Link>
-            </li>
-            <li>
-              <Link to={expensePage} data-cy="expense-link">
-                Expenses
-              </Link>
-            </li>
-          </ul>
+        <div className="usa-grid expenses-container">
+          <h3 className="expenses-header">Review Payment Request</h3>
+          <p>
+            {' '}
+            Make sure <strong>all</strong> your documents are uploaded.
+          </p>
+          <div>
+            <h3 style={{ color: 'red' }}>DOCUMENT SUMMARY TBU</h3>
+            <ul style={{ marginBottom: '30em' }}>
+              <li>
+                <Link to={weightTicketsPage} data-cy="weight-ticket-link">
+                  Weight Ticket
+                </Link>
+              </li>
+              <li>
+                <Link to={expensePage} data-cy="expense-link">
+                  Expenses
+                </Link>
+              </li>
+            </ul>
+          </div>
+          <div className="usa-grid" style={{ marginBottom: '4.5em' }}>
+            <CustomerAgreement
+              className="review-customer-agreement"
+              onChange={this.handleOnAcceptTermsChange}
+              link="/ppm-customer-agreement"
+              checked={this.state.acceptTerms}
+              agreementText={ppmPaymentLegal}
+            />
+          </div>
+          <PPMPaymentRequestActionBtns
+            nextBtnLabel={nextBtnLabel}
+            submitButtonsAreDisabled={!this.state.acceptTerms}
+            submitting={submitting}
+            displaySaveForLater={true}
```

Comment:
Could also just be this, accomplishes the same affect
```suggestion
            displaySaveForLater
```

Reply:
👍